bodhi/__init__.py

```diff
@@ -17,3 +17,6 @@
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 """Fedora's update manager."""
+
+# This is a regular expression used to match username mentions in comments.
+MENTION_RE = r'(?<!\S)(@\w+)'
```

Comment:
Well you could have `re.compile`'d the regexp right here and gained execution speed but that's really a detail ;-)

Reply:
Good idea, I'll make that so.

Reply:
I'm applying this patch before merging:

```
diff --git a/bodhi/__init__.py b/bodhi/__init__.py
index 9d79ab1d..d08279bb 100644
--- a/bodhi/__init__.py
+++ b/bodhi/__init__.py
@@ -18,5 +18,8 @@
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 """Fedora's update manager."""
 
+import re
+
+
 # This is a regular expression used to match username mentions in comments.
-MENTION_RE = r'(?<!\S)(@\w+)'
+MENTION_RE = re.compile(r'(?<!\S)(@\w+)')
```

Reply:
Actually I'm going to unapply that patch - ffmarkdown seems to really want text. Also, Python does cache recently used regular expressions, so I think we effectively get the same performance here anyway since we don't use any other regular expressions in the package.